nextjs_frontend/src/app/proposal/components/ProposalPodium/proposal-podium.module.scss

```diff
@@ -0,0 +1,30 @@
+.proposalCard {
+  @apply w-full h-96 mb-10 flex flex-col items-center justify-center
+    bg-white shadow border rounded-[10px] border-gray-950 border-opacity-50
+    transition-transform ease-in-out duration-300;
+
+  &.selected {
+    @apply scale-125 -translate-y-5;
+  }
+
+  ul {
+    @apply font-medium text-sm text-[#969696];
+
+    li {
+      @apply pl-7 py-2
+      bg-[url(/icons/check_circle.svg)] bg-no-repeat bg-left;
```

Comment:
What do you think to centralize the images on HTML?

Reply:
Yes

Reply:
I don't see any problems in doing it that way, but i think that in this case, adding the image as a style of the list makes the code cleaner